# -*- coding: utf-8 -*-
# Generated by Django 1.11 on 2017-04-16 22:56
from __future__ import unicode_literals

from django.db import migrations, models
import uuid


class Migration(migrations.Migration):

    dependencies = [
        ('authenticator', '0003_auto_20170416_2140'),
    ]

    operations = [
        migrations.RemoveField(
            model_name='authenticate_request',
            name='status',
        ),
        migrations.AddField(
            model_name='authenticate_data_request',
            name='code',
            field=models.UUIDField(default=uuid.uuid4, editable=False),
        ),
        migrations.AddField(
            model_name='authenticate_data_request',
            name='nombre',
            field=models.CharField(max_length=250, null=True),
        ),
        migrations.AddField(
            model_name='authenticate_data_request',
            name='response_datetime',
            field=models.DateTimeField(auto_now=True),
        ),
        migrations.AddField(
            model_name='authenticate_data_request',
            name='status',
            field=models.IntegerField(choices=[(1, 'Solicitud recibida correctamente.'), (2, 'Ha ocurrido algún problema al solicitar la firma.'), (3, 'Solicitud con campos incompletos.'), (4, 'Diferencia no permitida de hora entre cliente y servidor.'), (5, 'La entidad no se encuentra registrada.'), (6, 'La entidad se encuentra en estado inactiva.'), (7, 'La URL no pertenece a la entidad solicitante.'), (8, 'El tamaño de hash inválido debe estar entre 1 y 130 caracteres.'), (9, 'Algoritmo desconocido'), (10, 'Certificado incorrecto')], default=1),
        ),
    ]
